webapp/actions/post_actions.jsx

```diff
@@ -500,10 +303,27 @@ export function performSearch(terms, isMentionSearch, success, error) {
     );
 }
 
-export function storePostDraft(channelId, draft) {
-    AppDispatcher.handleViewAction({
-        type: ActionTypes.POST_DRAFT_CHANGED,
-        channelId,
-        draft
-    });
+let loadingMorePosts = false;
+
+// Returns true if there are no more posts to load
```

Comment:
Not sure but we're not using that convention anywhere I made a comment specifically stating the behaviour